internal/rebac_admin/resources.go

```diff
@@ -58,3 +52,17 @@ func (s *resourcesService) ListResources(ctx context.Context, params *resources.
 		},
 	}, nil
 }
+
+// We fetch one record more than the page size.
+// Then, we set the next page if we have this many records.
+// If it does we return the records minus 1 and advide the consumer there is another page.
```

Comment:
The wording is confusing here,
Reading this I see
- If we have as many records as requested, we set the next page.
- If we don't have as many records as request, we return records minus 1 and tell client there is another page.
- Else we return the records and set the next page as empty.


Aren't points 1 and 3 saying the same thing?

Reply:
you are right, i made a mess with c/p. Now it's updated.